Code/LauncherUnified/CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ ly_install_directory(
         Platform/${PAL_PLATFORM_NAME}
         Platform/Common
     DESTINATION LauncherGenerator/Platform
+    VERBATIM 
```

Comment:
The project generation for game projects depends on cmake files in LauncherUnified's platform directory.